### PR TITLE
done hw

### PR DESCRIPTION
**Pull request by Alex-Korzun**

### CODE REVIEW

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+
+    public static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:7]:
```suggestion
    public static final int COUNT_OF_FIGURES = 5;
```
Figure does not have implamentation, cause it's a class

File: src/main/java/core/basesyntax/Area.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    public double calculateArea();
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Area.java:4]:
```suggestion
    double calculateArea();
```
methods in an interface are public abstract by default

File: src/main/java/core/basesyntax/Draw.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    public void draw();
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/Draw.java:4]: same here

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]
good job left minor comments

File: src/main/java/core/basesyntax/Draw.java (hunk 3 above)
Comment (Alex-Korzun) [reply src/main/java/core/basesyntax/Draw.java]: done

File: src/main/java/core/basesyntax/FigureProducer.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+
+    public static final int COUNT_OF_FIGURE = 5;
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/FigureProducer.java:7]:
```suggestion
    private static final int COUNT_OF_FIGURE = 5;
```
if we use some element only once in the class we should keep it as strict as possible

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request closed without merging**